pyincore_viz/plotutil.py

```diff
@@ -9,11 +9,13 @@
 import numpy
 import pandas as pd
 
+from deprecated.sphinx import deprecated
+
 
 class PlotUtil:
     """Plotting utility."""
 
-    @staticmethod
+    @deprecated(version="1.8.0", reason="It is not being used anymore. Check get_x_y or get_x_y_z")
```

Comment:
Do you know if there are any notebooks of ours still using this method?

Reply:
let me check

Reply:
Just checked this too. I could not find a reference to this method across incore-docs, and it would be safe to assume that any person using a version of it with these methods must update versions of the IN-CORE toolchain as well.

Reply:
i don't think @deprecated decorator will actually prevent this method from working; it will just show the message